### PR TITLE
Removing special character from commands documentation

### PR DESCRIPTION
**Pull request by gilMars**
Special character in documentation is inducing the error of copying and pasting.

### CODE REVIEW

File: docs/mlgit_commands.md [hunk 1]
```diff
@@ -816,7 +816,7 @@ Options:
 
 Example:
 ```
-$ ml-git repository config show
+ml-git repository config show
```
Comment (adysonl) [inline docs/mlgit_commands.md:819]: it was also suggested to isolate the commands (separating them from outputs, inputs etc)

Comment (gilMars) [reply docs/mlgit_commands.md:828]: Adjusted in 4f996fd. Thanks a lot.

**Review by talessatiro — APPROVED** [review]

File: docs/aws_s3_configuration.md [hunk 2]
```diff
@@ -32,34 +32,31 @@ You can configure the AWS in three ways (environment variables, through the cons
...
    **Linux or macOS**:
 
-    ```
-    $ export AWS_ACCESS_KEY_ID=your-access-key
-    $ export AWS_SECRET_ACCESS_KEY=your-secret-access-key
-    $ export AWS_DEFAULT_REGION=us-west-2
-    ```
+    export AWS_ACCESS_KEY_ID=your-access-key
+    export AWS_SECRET_ACCESS_KEY=your-secret-access-key
+    export AWS_DEFAULT_REGION=us-west-2
```
Comment (andersonpablosilva) [inline docs/aws_s3_configuration.md]: Please, add mark quotes here and indent the text (apply this to the entire section).

Comment (gilMars) [reply docs/aws_s3_configuration.md]: The mark quotes added and the indentation adjusted in d8a958c.

**Review by andersonpablosilva — APPROVED** [review]

File: docs/first_steps.md [hunk 3]
```diff
@@ -89,13 +89,13 @@ Corrupted files:
...
+
+It will add all untracked files
 ```
-# It will add all untracked files
 ml-git datasets add imagenet8
 ```
-
+ It will commit the metadata to the local repository
 ```
-# It will commit the metadata to the local repository
```
Comment (andersonpablosilva) [inline docs/first_steps.md:98]: Please, revert this change. I think we could keep these comments in the code snippets.

File: docs/aws_s3_configuration.md [hunk 4]
```diff
@@ -30,65 +30,64 @@ You can configure the AWS in three ways (environment variables, through the cons
...
-   aws_secret_access_key = your-secret-access-key     
-   " > .aws/credentials
-   ```
+From the home directory (UserProfile) execute:   
+  
+```
+mkdir .aws
+```
+
+You need to create two files to store the sensitive credential information (~/.aws/credentials) separated from the less sensitive configuration options (~/.aws/config). To create these two files type the following commands:
```
Comment (andersonpablosilva) [inline docs/aws_s3_configuration.md:54]:
```suggestion
You need to create two files to store the sensitive credential information (\~/.aws/credentials) separated from the less sensitive configuration options (\~/.aws/config). To create these two files type the following commands:
```

File: docs/minio_s3_configuration.md [hunk 5]
```diff
@@ -45,8 +45,8 @@ You can configure the credentials in three ways (environment variables, through
    **Linux or macOS**:
 
     ```
```
Comment (andersonpablosilva) [inline docs/minio_s3_configuration.md:47]: Please, remove unnecessary indentations in this file.

File: docs/first_steps.md (hunk 3 above)
Comment (gilMars) [reply docs/first_steps.md:98]: In my opinion, the text could be adjusted and maintain it out of code block for readability purposes.

File: docs/aws_s3_configuration.md (hunk 4 above)
Comment (gilMars) [reply docs/aws_s3_configuration.md]: Adjusted in 3126b78.

File: docs/minio_s3_configuration.md (hunk 5 above)
Comment (gilMars) [reply docs/minio_s3_configuration.md]: Adjusted in 3126b78.

**Review by andersonpablosilva — APPROVED** [review]
